### ISSUE TITLE
Problem with buckets in US Standard region

### ISSUE THREAD

**Issue by antranapp** [aws/aws-sdk-php#18]
I created a bucket in US Standard region. Initialize the s3client using Region::US_EAST_1 . When trying to upload a file, i received this error "The request signature we calculated does not match the signature you provided. Check your key and signing method."

Change Region to Region::US_WEST_1, i received this error: The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.

Now i created a bucket in Ireland region and intialize s3Client using Region::EU_WEST_1, everything works fine. 

i created a bucket in North California region and intialize s3Client using Region::US_WEST_1, everything works fine. 

i created a bucket in Oregon region and intialize s3Client using Region::US_WEST_2, everything works fine. 

So basically, i have problem with bucket created in US Standard region
